monai/handlers/ignite_metric.py

```diff
@@ -35,7 +36,7 @@
     )
 
 
-class IgniteMetric(Metric):
```

Comment:
To provide a deprecation warning about the old name I would create a new subclass of `IgniteMetricHandler` called `IgniteMetric` and decorate this with [deprecated](https://github.com/Project-MONAI/MONAI/blob/17c1e3acff583e241066da08dae4b75554b52dda/monai/utils/deprecate_utils.py#L41). The `since` value should be "1.3" and `removed` should be "1.4". The `msg_suffix` should mention the new name. You can search the repo for examples of using this decorator with functions and classes.